index.html

```diff
@@ -1891,7 +1893,180 @@ <h4>Create Workflow</h4>
           data-api-endpoint="post /workflows"></div>
 
         <p>
-For the `openId` options above, specifically the values
+Most workflows will contain information about steps, credential templates,
+and authorization. In some cases, a workflow will also reference the
+reserved `results` variable at different points in its configuration
+(typically in credential templates). This variable holds data that
+the workflow service receives from clients throughout the course
+of an exchange. Here is a basic workflow configuration that contains
+a DID Authentication step and a credential delivery step:
```

Comment:
@dlongley Do you think this is a decent enough explanation/motivation for the `results` variable given our discussions in this PR and elsewhere?